examples/gmd_aquaplanet.py

```diff
@@ -0,0 +1,113 @@
+import climt
+from sympl import (
+    PlotFunctionMonitor, NetCDFMonitor,
+    TimeDifferencingWrapper, UpdateFrequencyWrapper,
+    DataArray
+)
+import numpy as np
+from datetime import timedelta
+
+
+def plot_function(fig, state):
```

Comment:
Not for right now, but we should consider making some more plot monitors that are initialized easier (e.g. just by telling it what quantities to plot, and it figures out how to do it from the data dimensions).

Reply:
Yes, I have been thinking about this for some time. Don't have much time in the near future to attempt this, but it would definitely be a useful addition.